src/pubvis.js

```diff
@@ -535,6 +539,37 @@ PUBVIS = function () {
                 //console.log( "keywords.length: " + keywords.length );
             }
 
+            exclude_chosen_keywords = function (keywords) {
+                if(exclude_chosen_keywords.length > 0) {
```

Comment:
1. not sure if `exclude_chosen_keywords` is a typo instead of `exclude_keywords`
2. testing if `exclude_keywords` is an array and thus not `undefined` improves the backwards compatibility.  
Thus I would change this line to:
`if(Array.isArray(exclude_keywords) && exclude_keywords.length > 0) {`